ros/content.md

```diff
@@ -1,28 +1,116 @@
-# What is [ROS](http://www.ros.org/)?
+# What is [ROS](https://www.ros.org/)?
```

Comment:
As these docs now default to ROS2, we could use this link instead http://www.ros2.org/ (which redirects to https://index.ros.org/doc/ros2/)

Reply:
Is that going to be the new landing page?
I just figured the top most link should point to the top most domain.

Reply:
I'm going to keep this as is. We can revisit this when updating the docs adding the ros1-bridge tag.